base/event.jl

```diff
@@ -127,8 +127,8 @@ function schedule(t::Task, arg; error=false)
     return enq_work(t)
 end
 
-# fast version of schedule(t,v);wait()
-function schedule_and_wait(t::Task, v=nothing)
+# fast version of `schedule(t, arg); wait()`
+function schedule_and_wait(t::Task, arg=nothing)
```

Comment:
`v=nothing` or change `v` to `arg` in the body.

And a test for `schedule_and_wait`